.travis.yml

```diff
@@ -13,6 +13,6 @@ addons:
   # CI for now.
   postgresql: "9.2"
 
-before_script: bundle exec rake db:create
+before_script: RAILS_ENV=test bundle exec rake db:create
 
```

Comment:
I think this environment variable is set by default by Travis: https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables